solutions/usaco-940.mdx

```diff
@@ -2,11 +2,48 @@
 id: usaco-940
 source: USACO Bronze 2019 Open
 title: Milk Factory
-author: Mrinall Umasudhan and Jesse Choe
+author: Mrinall Umasudhan, Jesse Choe, Ryan Chou
 ---
 
+## Solution 1
 [Official Analysis](http://www.usaco.org/current/data/sol_factory_bronze_open19.html)
 
+## Implementation
+<LanguageSection>
+<PySection>
+```py
+
+fin = open('factory.in', 'r')
+
+N = int(fin.readline())
+
+outgoing = [0] * 101
+incoming = [0] * 101
```

Comment:
magic numbers are kinda bad
1. are you sure you need statically sized arrays
2. if you 100% need them, put it in a constant called like `MAX_[smth]` idk

Reply:
👍